### ISSUE TITLE
"トゥドゥドェ" seems unsupported

### ISSUE THREAD

**Issue by corrupt952** [makimoto/romaji#10]
Hi, @makimoto.
I found some kana spellings were not correctly recognized.

```ruby
> Romaji.kana2romaji('トゥエンティーフォー')
=> "toゥenthi-fo-"
> Romaji.kana2romaji('ドゥードゥー')
=> "doゥ-doゥ-" 
> Romaji.kana2romaji('ドェントナ')
=> "doェntona"
```

**Comment by makimoto** [comment]
Thank you for reporting it.
I fixed it in d7e6edb9751a140f0ff0f44c6bc8134678723335 and released [v2.2.3](https://rubygems.org/gems/romaji/versions/0.2.3).

**Comment by corrupt952** [comment]
Thanks, @makimoto .
